### ISSUE TITLE
assess sapt-calling names

### ISSUE THREAD

**Issue by loriab** [psi4/psi4#1835]
right now, one gets (wfn) sapt0-inf through sapt(dft). in light of that new addition, plus outright sapt(dft) plus sapt0-d, take a wholistic looks at the arguments to `energy(sapt*)`

**Comment by CDSherrill** [comment]
Yeah we should sit down and try to sort out all the latest SAPT additions
and how they should be called.

**Comment by JonathonMisiewicz** [comment]
While not explicitly discussed on conference call, this is the kind of quality-of-life improvement that seems like a better 1.5 target.

**Comment by hokru** [comment]
Because I wanted to access certain variants I did some work towards this on a local branch.

**Comment by loriab** [comment]
I think consolidation, reshuffling, and extension of the SAPT trio is a spring project.
